docker/docker-compose.2404.yaml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+
+services:
+
+  test-setup:
```

Comment:
are there Dockerfiles for this?

Reply:
I believe that the docker files for these are covered by the images from docker hub, like `swift:5.10-noble`, and also the `test.dockerfile` and `install-test.dockerfile` which are ubuntu-based with some variables for the ubuntu version. These Dockerfiles should be invoked by the CI, but I don't know the magic hooks to make the CI try these new platforms now that we have the docker and docker compose files. Maybe @shahmishal can hook up CI for them?

Reply:
In this latest version of the PR none of these are needed anymore since we switched to GH actions. The docker setup is managed by the official docker containers by the reusable workflow in `swiftlang/github-workflow`.